docs/source/algorithms/SaveNexusGeometry-v1.rst

```diff
@@ -10,35 +10,60 @@
 Description
 -----------
 
-TODO: Enter a full rst-markup description of your algorithm here.
+Saves the Instrument geometry from the workspace to a Nexus file. Saves detector positions and rotations.
+Current version (1.0) does NOT save instrument shapes of instrument components. file is compliant to Nexus standards.
+For more information on the Nexus format, see https://www.nexusformat.org/
+
+the Instrument will be extracted from the specified workspace, and written to the specified location.
+
+The (optional) H5 root group name is the parent group in which the Instrument and sample data are stored.
+If no name is given, the root group will have a default nme of 'entry'
 
 
 Usage
 -----
-..  Try not to use files in your examples,
-    but if you cannot avoid it then the (small) files must be added to
-    autotestdata\UsageData and the following tag unindented
-    .. include:: ../usagedata-note.txt
 
 **Example - SaveNexusGeometry**
 
 .. testcode:: SaveNexusGeometryExample
 
-   # Create a host workspace
-   ws = CreateWorkspace(DataX=range(0,3), DataY=(0,2))
-   or
-   ws = CreateSampleWorkspace()
+   # create sample workspace
+ws1 = CreateSampleWorkspace()
+instr1 = ws1.getInstrument()
+compInfo = ws1.componentInfo()
+detInfo = ws1.detectorInfo()
+
+   # save Instrument in workspace with default H5 root name 'entry'
+print("===========================")
+#SaveNexusGeometry(workspace=ws1, FileName1="somePathToFile.nxs")
+FileName1 = "path/to/someTestFile.nxs"
+#SaveNexusGeometry(workspace=ws2, FileName="somePathToFile", H5Path="testRoot")
+print("the Instrument {0} was saved to location {1}".format(instr1.getName(), FileName1))
+
+  # save instrument in workspace with specified H5 root group name
+print("===========================")
+ws2 = CreateSampleWorkspace();
+instr2 = ws2.getInstrument()
+detInfo2 = ws2.detectorInfo()
+compInfo2 = ws2.componentInfo()
 
-   wsOut = SaveNexusGeometry()
+FileName2 = "path/to/someTestFile.hdf5"
+#SaveNexusGeometry(workspace=ws2, FileName="somePathToFile", H5Path="testRoot")
+print("the Instrument {0} was saved to location {1}".format(instr2.getName(), FileName2))
 
-   # Print the result
-   print "The output workspace has %%i spectra" %% wsOut.getNumberHistograms()
 
 Output:
 
 .. testoutput:: SaveNexusGeometryExample
 
-  The output workspace has ?? spectra
+	CreateSampleWorkspace started
+	CreateSampleWorkspace successful, Duration 0.00 seconds
+	===========================
+	the Instrument basic_rect was saved to location path/to/someTestFile.nxs
+	===========================
```

Comment:
Avoid these ruler lines, if you want to include multiple samples you can include multiple code and output blocks.